### PR TITLE
PR for flaring stars

### PR DESCRIPTION
**Pull request by mcoughlin**
I put a quick PR together for the flares.

### CODE REVIEW

**Review by przemekmroz — COMMENTED** [review]
The first three objects from your golden sample are actually dwarf novae, not flaring stars. Do I remember correctly that in the phenomenological classification a "flaring star" means any kind of outbursting source? (e.g., a flaring M dwarf, dwarf nova, etc)? If so, you need to change the description. If not, example flaring red dwarfs are: 
ra dec
103.289525 10.194726
289.148993 -6.298941
304.720578 36.500362
315.439236 49.800311
286.523041 -16.346877
269.983918 -22.852188
288.186894 9.998263
281.757732 29.648821
285.342961 27.499306
292.879252 6.336936

**Review by przemekmroz — COMMENTED** [review]
Looks good. Can you add a light curve of a flaring M dwarf?

dwarf novae (U Geminorum-type variable stars) which
--> dwarf novae which 
[dwarf novae include U Gem and SU UMa-type variable stars]

**Review by przemekmroz — APPROVED** [review]

**Comment by dmitryduev** [comment]
@AshishMahabal is this ready to be merged?

**Comment by dmitryduev** [comment]
@mcoughlin do you want to add the sky position plots here?

File: doc/field_guide__flaring.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Flaring variables (flaring)
+
+The phenomelogical classification of ``flaring`` is any kind of outbursting source, where a variable star (or binary system) dramatically increases in brightness for minutes to hours before returning to quiescence. Most flare stars are red dwarfs, while RS Canum Venaticorum variables (RS CVn) are also known to flare due to a companion star in the binary system. Other possible sources of flaring include dwarf novae which are increases in luminosity in the accretion disk of a cataclysmic variable star.
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:3]: Reword. Dwarf novae are not an increase in luminosty. "... increases are ..." also do not juxtapose well.

File: doc/field_guide__flaring.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Flaring variables (flaring)
+
+The phenomelogical classification of ``flaring`` is any kind of outbursting source, where a variable star (or binary system) dramatically increases in brightness for minutes to hours before returning to quiescence. Most flare stars are red dwarfs, while RS Canum Venaticorum variables (RS CVn) are also known to flare due to a companion star in the binary system. Other possible sources of flaring include dwarf novae which are increases in luminosity in the accretion disk of a cataclysmic variable star.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - flaring
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:8]: Can not be its own supertype

File: doc/field_guide__flaring.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+## Flaring variables (flaring)
+
+The phenomelogical classification of ``flaring`` is any kind of outbursting source, where a variable star (or binary system) dramatically increases in brightness for minutes to hours before returning to quiescence. Most flare stars are red dwarfs, while RS Canum Venaticorum variables (RS CVn) are also known to flare due to a companion star in the binary system. Other possible sources of flaring include dwarf novae which are increases in luminosity in the accretion disk of a cataclysmic variable star.
+
+### Classification and numbers
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:5]: Add subtypes and expected numbers.

File: doc/field_guide__flaring.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  - flaring
+
+
+### ZTF light curves
+![ZTF flaring_dwarfnova](data/flaring_dwarfnova.png)
+
+![ZTF flaring_mdwarf](data/flaring_mdwarf.png)
+
+#### Description
+Flaring stars are easy to recognise by their dramatic increase in brightness from quiescent levels, rapidly returning to that same level shortly thereafter.
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:17]: 'drastic' is very qualitative. Some example quantities would be useful.

File: doc/field_guide__flaring.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+#### Light curve characteristics
+- variable
+- dramatic increases in brightness
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:21]: again, some quantification. Dramatic for one can be puny for another.

File: doc/field_guide__flaring.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+![HR diagram of Flaring](data/hr__flaring.png)
+
+![RA/Dec diagram of Flaring](data/radec__flaring.png)
+
+### References and further reading:
+- Sterken & Jasschek: Light curves of variable stars
```
Comment (AshishMahabal) [inline doc/field_guide__flaring.md:28]: Link or fully resolved reference?

File: doc/field_guide__flaring.md (hunk 3 above)
Comment (mcoughlin) [reply doc/field_guide__flaring.md:5]: I added relative occurrence rates for what I understand about red dwarfs.

**Review by AshishMahabal — APPROVED** [review]

**Comment by mcoughlin** [comment]
@AshishMahabal you should be able to merge if you are happy.

**Pull request merged**